bin/setup

```diff
@@ -46,6 +46,9 @@ FileUtils.chdir APP_ROOT do
     system! "bin/rails db:setup"
   end
 
+  puts "\n== Updating Fastly =="
+  system! "bin/rails fastly:update_whitelisted_params"
```

Comment:
What's the reason for having this here since the rake task will only ever run in production?

Reply:
I thought this is how we'd run the rake task on deployment. Where should it be moved to?